packages/core/src/shared/globalState.ts

```diff
@@ -46,6 +46,7 @@ export type globalKey =
     | 'aws.toolkit.amazonqInstall.dismissed'
     | 'aws.amazonq.workspaceIndexToggleOn'
     | 'aws.amazonq.customization.override'
+    | 'aws.amazonq.customization.overrideV2'
```

Comment:
it should be safe to to remove L48 here I think `'aws.amazonq.customization.override'`

Reply:
ok